glustercli/cmd/common.go

```diff
@@ -56,6 +60,11 @@ func failure(msg string, err error, errcode int) {
 
 	w.WriteString(msg + "\n")
 
+	if client == nil && err != nil {
```

Comment:
question: when the client will become `nil`?

Reply:
if we failed to set up (initialize) client, in that case client will be nil